docs/node/migrations/zero-downtime-migrations/index.mdx

```diff
@@ -122,4 +122,4 @@ Conflict resolution principles for account spends:
 
 ## Migration Ends
 
-The old network halts after the `Duration` blocks since the start of the migration. The old network will no longer accept new transactions or produce new blocks with transactions. The RPC services remain available for querying the state of the network. Once the new network is fully synced with the old network, the old network can be safely decommissioned. The new network continues to process transactions and produce blocks as usual.
\ No newline at end of file
+The old network halts after the `Duration` blocks since the start of the migration. The old network will no longer accept new transactions or produce new blocks with transactions. The RPC services remain available for querying the state of the network. Once the new network is fully synced with the old network, the old network can be safely decommissioned. The new network continues to process transactions and produce blocks as usual.
```

Comment:
I used an online diff checker to ensure that these are the same. IDK why git is saying these are different